packages/ui/src/components/va-slider/VaSlider.vue

```diff
@@ -62,8 +62,8 @@
           class="va-slider__handler"
           :class="dotClass[order]"
           :style="dottedStyles[order]"
-          @mousedown="(moveStart($event, order), hasMouseDown = true)"
-          @touchstart="moveStart($event, order)"
+          @mousedown="moveStart($event, order), hasMouseDown = true"
```

Comment:
Shouldn't we set `hasMouseDown` inside `moveStart`? Didn't we miss it here too?

https://github.com/epicmaxco/vuestic-ui/blob/fdc9fc4198a3cd8cc718e1a26414760c12e3945f/packages/ui/src/components/va-slider/VaSlider.vue#L96